public/locales/en/translation.json

```diff
@@ -655,13 +655,20 @@
         "linkAccount": "Link your Steam account to check eligibility.",
         "needMoreAchievements": "You need to have completed {{percent}}% of the achievements in one of these games.",
         "Play": "Play",
-        "playstreak": "Play Streak",
+        "playstreak": {

```

Comment:
Where those translations generated using the `yarn i18n` command?
I got a different result from it:
![image](https://github.com/user-attachments/assets/211dfe51-d1d2-4510-bee4-e78a131e9e40)

It is important to use this command since Weblate uses the same pattern as it to update the translations.

Reply:
yes, but I think I forgot to run it after fixing those type keys. 🙏 Will run and push now